frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewMetaData/FeatureOverviewMetaData.tsx

```diff
@@ -97,55 +79,46 @@ const FeatureOverviewMetaData = () => {
 
     const showDependentFeatures = useShowDependentFeatures(project);
 
-    const FlagTypeIcon = getFeatureTypeIcons(type);
-
     return (
         <>
             <StyledMetaDataContainer>
-                <StyledMetaDataHeader data-loading>
-                    <FlagTypeIcon />
-                    <h2>{capitalize(type || '')} flag</h2>
-                </StyledMetaDataHeader>
-                <ConditionallyRender
-                    condition={Boolean(description)}
-                    show={
+                <div>
+                    <StyledTitle>Flag details</StyledTitle>
+                    {description ? (
                         <StyledMetaDataItem data-loading>
                             <StyledMetaDataItemText>
                                 {description}
                             </StyledMetaDataItemText>
                         </StyledMetaDataItem>
-                    }
-                />
+                    ) : null}
```

Comment:
Same question regarding ternaries vs double ampersands. Wouldn't `&&` be easier? But did we say not to use double ampersands?